src/NServiceBus.Core.Tests/App.config

```diff
@@ -1,39 +1,43 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <configSections>
-    <section name="TestConfigurationSection" type="NServiceBus.Core.Tests.Config.TestConfigurationSection, NServiceBus.Core.Tests"/>
-    <section name="UnicastBus_with_empty_ttr" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core"/>
-    <section name="UnicastBus_with_ttr_set" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core"/>
-    <section name="GatewayConfig" type="NServiceBus.Config.GatewayConfig, NServiceBus.Core"/>
+    <section name="TestConfigurationSection" type="NServiceBus.Core.Tests.Config.TestConfigurationSection, NServiceBus.Core.Tests" />
+    <section name="UnicastBus_with_empty_ttr" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
+    <section name="UnicastBus_with_ttr_set" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
+    <section name="GatewayConfig" type="NServiceBus.Config.GatewayConfig, NServiceBus.Core" />
   </configSections>
-  <TestConfigurationSection TestSetting="test"/>
+  <TestConfigurationSection TestSetting="test" />
 
-  <UnicastBus_with_empty_ttr/>
+  <UnicastBus_with_empty_ttr />
 
-  <UnicastBus_with_ttr_set TimeToBeReceivedOnForwardedMessages="00:30:00"/>
+  <UnicastBus_with_ttr_set TimeToBeReceivedOnForwardedMessages="00:30:00" />
 
   <appSettings>
-    <add key="ListenUrl" value="http://localhost:8090/Gateway/"/>
-    <add key="RemoteUrl" value="http://localhost:8092/Gateway/"/>
+    <add key="ListenUrl" value="http://localhost:8090/Gateway/" />
+    <add key="RemoteUrl" value="http://localhost:8092/Gateway/" />
   </appSettings>
 
   <GatewayConfig>
     <Sites>
-      <Site Key="SiteA" Address="http://sitea.com" ChannelType="Http"/>
+      <Site Key="SiteA" Address="http://sitea.com" ChannelType="Http" />
     </Sites>
     <Channels>
-      <Channel Address="http://localhost/headquarter" ChannelType="Http"/>
-      <Channel Address="ftp://localhost:21" ChannelType="ftp"/>
-      <Channel Address="https://localhost/headquarter" ChannelType="Http" NumberOfWorkerThreads="3"/>
+      <Channel Address="http://localhost/headquarter" ChannelType="Http" />
+      <Channel Address="ftp://localhost:21" ChannelType="ftp" />
+      <Channel Address="https://localhost/headquarter" ChannelType="Http" NumberOfWorkerThreads="3" />
     </Channels>
   </GatewayConfig>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-      <probing privatePath="bin;TestDlls"/>
+      <probing privatePath="bin;TestDlls;assemblyscannerfiles" />
```

Comment:
yes it is for the probing path in order for the 

`AppDomain.CurrentDomain.ApplyPolicy(assemblyNameFullName);`

to be successful